core/src/test/java/kafka/admin/ConfigCommandIntegrationTest.java

```diff
@@ -138,6 +139,36 @@ public void testNullStatusOnKraftCommandAlterClientMetrics() {
         assertEquals("Completed updating config for client-metric cm.", message);
     }
 
+    @ClusterTest
+    public void testAddConfigKeyValuesUsingCommand() throws Exception {
+
+        try (Admin client = cluster.admin()) {
+            NewTopic newTopic = new NewTopic("test-topic", 1, (short) 1);
+            client.createTopics(Collections.singleton(newTopic)).all().get();
+            cluster.waitForTopic("test-topic", 1);
+
+            Stream<String> command = Stream.concat(quorumArgs(), Stream.of(
+                    "--entity-type", "topics",
+                    "--entity-name", "test-topic",
+                    "--alter", "--add-config", "cleanup.policy=[delete,compact]"));
+
+            String message = captureStandardStream(false, run(command));
+            assertEquals("Completed updating config for topic test-topic.", message);
+
+            command = Stream.concat(quorumArgs(), Stream.of(
+                    "--entity-type", "topics",
+                    "--entity-name", "test-topic",
+                    "--describe"));
+
+            message = captureStandardStream(false, run(command));
+            assertTrue(message.contains("cleanup.policy=delete,compact"), "Config entry was not added correctly");
+
+            //Clean up by deleting topic
+            client.deleteTopics(Collections.singleton("test-topic")).all().get();
```

Comment:
We also need to make sure topic already deleted here.
`cluster.waitForTopic("test-topic, 0);`

Reply:
Yes,you are right. Thanks!

Reply:
> We also need to make sure topic already deleted here. `cluster.waitForTopic("test-topic, 0);`

Hi,I find that the topic will be deleted automatically,so may be there is no need to do that manually?

Reply:
You are right. `clusterInstance` will delete dir automatically.